.github/workflows/push.yml

```diff
@@ -134,3 +137,57 @@ jobs:
         env:
           TARGET_VERSION: ${{ needs.set_version.outputs.version }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  release-macos:
+    needs:
+      - set_version
+      - cross-compile
+      - tests
+    runs-on: macos-latest
+    if: startsWith(github.ref, 'refs/tags/') && needs.set_version.outputs.is_semantic_tag == 'true'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Download-Binaries
+        uses: actions/download-artifact@v2
+        with:
+          name: darwin-binaries
+          path: release/
+      - name: Prepare files
+        run: |
+          mkdir release/signed
+          mkdir release/darwin-amd64
+          cp release/*amd64/kbc release/darwin-amd64
+          mkdir release/darwin-arm64
+          cp release/*arm64/kbc release/darwin-arm64
```

Comment:
kopirujem si do zvlastnych zloziek `release/darwin-arm64` resp `amd64` pretoze v gon.json configu musi byt pevna cesta, po release je dynamicka(obsahuje verziu) a to do gon.json nemozem dat